bundles/org.openhab.binding.atlona/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,20 @@
 	<description>Binding for Atlona PRO3 HDBaseT Matrix switches.</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>sddp</service-type>
+			<match-properties>
```

Comment:
No need to consider primaryProxy ?

Reply:
I think Manufacturer + Model should be sufficient